sdk/customers.go

```diff
@@ -11,3 +14,19 @@ func ValidateCustomers() bool {
 	}
 	return true
 }
+
+//ValidateCustomerList validate customer names list
+func ValidateCustomerList(customers []string) bool {
+	for i, customerName := range customers {
+		for j, cn := range customers {
+
+			if i != j {
+				if strings.HasPrefix(cn, customerName+"-") {
+					return false
```

Comment:
i think it should also return a error msg specifying which username(s) caused this validation to fail and if possible, we should expose that in the PR automatically.

Reply:
That's a good point. Perhaps we can change this to return the names which conflict @viveksyngh for debugging and better diagnostics?